hologres/README.md

```diff
@@ -0,0 +1,20 @@
+Hologres is a HSAP (Hybrid Serving/Analytics Processing) real-time data warehouse, unified for both PS-scale real-time data analytics and serving data for fast point-lookups.
```

Comment:
Should that mean "PB-scale" (peta byte)?

Reply:
Thank you for catching that mistake. It should be "PB-scale". I will update the description.